### PR TITLE
Update GetVehicleFlightNozzlePosition examples for use up to date method name

### PR DESCRIPTION
**Pull request by Svetch**
Update examples for use up to date method name

### CODE REVIEW

**Comment by ahcenezdh** [comment]
You could add a JS example if you want.

**Comment by Svetch** [comment]
I added a JS example.

**Review by 4mmonium — APPROVED** [review]
LGTM, thank you for your contribution.

File: VEHICLE/GetVehicleFlightNozzlePosition.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ A float indicating the percentage of the hover mode. 1.0 = in VTOL mode, 0.0 = i
 ```lua
 local vehicle = GetVehiclePedIsIn(PlayerPedId(), false)
```
Comment (OneMajorFam) [inline VEHICLE/GetVehicleFlightNozzlePosition.md:20]: Trackhawk

File: VEHICLE/GetVehicleFlightNozzlePosition.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ end
 ```cs
 int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
 
-if (GetVehicleHoverModePercentage(vehicle) == 0f)
+if (GetVehicleFlightNozzlePosition(vehicle) == 0f)
```
Comment (OneMajorFam) [inline VEHICLE/GetVehicleFlightNozzlePosition.md:35]: New car 2023

**Review by OneMajorFam — APPROVED** [review]

**Pull request merged**